blocks/activity-list/index.js

```diff
@@ -58,23 +58,21 @@ export default registerBlockType(
 					<div className='max-width-3 mx-auto px1 md-px2'>
 						<h3 className='bold h1 line-height-2'>{ heading }</h3>
 					</div>
-					<div className='overflow-scroll'>
```

Comment:
This diff also looks more extensive than it is. I simply deleted the line above and its closing `</div>`, and re-indented its child elements.